pkg/model/request/request.go

```diff
@@ -0,0 +1,187 @@
+// Package request contains plain old Go types used in the Gin endpoint handlers
+// and Swaggo documentation for the HTTP request models, with Gin- and
+// Swaggo-specific Go tags.
+//
+// Copied from github.com/iver-wharf/wharf-api/pkg/model/request
```

Comment:
nit: Would be nice with a version reference. Example:

```suggestion
// Copied from https://github.com/iver-wharf/wharf-api/blob/v5.0.0/pkg/model/request/request.go
```